apps/api/src/app/inbox/usecases/mark-many-notifications-as/mark-many-notifications-as.usecase.ts

```diff
@@ -260,7 +260,7 @@ function createTraceLog({
     entity_type: 'step_run',
     entity_id: message._jobId,
     step_run_type: message.channel as StepType,
-    workflow_run_identifier: message.templateIdentifier,
```

Comment:
Is that intentional?

Reply:
yes, this mapping is incorrect.
sorry i wanted to create a comment about it.